backend-processing.ts

```diff
@@ -53,74 +32,86 @@ export default class BackendProcessing {
         console.log("chunk: "+chunk);
         return chunk;
       }});
-    console.log("result: "+JSON.stringify(result))
-    await results.data.forEach(element => {
-      if (element["name"] != null) this.call_from_csv_line(element, con);
-    });
-    this.end_connection(con);
-  }
-
-  async call_from_csv_line(element, con) {
-    let pcall = this.createCallForCSV(element);
-    if (pcall != null) {
-      console.log(`Sending to db: ${pcall}`);
-      con.query(pcall, (err, rows) => {
-        if (err) console.log("error from database: " + err);
-        else console.log(rows);
+      await results.data.forEach(element => {
```

Comment:
We have a known bug that the frontend can't autorefresh after uploading a csv. In hindsight, this might be a place to use the Promise.all statement you and Sam recommended.